plugins/retry/src/main/java/forklift/retry/RetryHandler.java

```diff
@@ -47,8 +49,9 @@ public RetryHandler(ForkliftConnectorI connector) {
     public RetryHandler(ForkliftConnectorI connector, File dir) {
         this.dir = dir;
         this.connector = connector;
-        this.mapper = new ObjectMapper();
-        this.mapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
+        this.mapper = new ObjectMapper().registerModule(new JavaTimeModule())
+                                        .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+                                        .configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
```

Comment:
This was a weird one.  Even though my POJO had the JSON Ignore annotation, it refused to honor it and would through the a warning and exception.   This was the only way I could fix it.